### ISSUE TITLE
Error "Unable to launch language server: ENOENT: no such file or directory" with version 2.15.0

### ISSUE THREAD

**Issue by cezarsa** [hashicorp/vscode-terraform#802]
## Versions

This bug is reproducible in:
 - [X] the **latest** version of the extension (below)

### Extension

```
HashiCorp Terraform
v2.15.0
```

### VS Code

```
Version: 1.60.2
Commit: 7f6ab5485bbc008386c4386d08766667e155244e
Date: 2021-09-22T11:59:08.826Z (1 wk ago)
```

### Operating System

```
Mac OS Big Sur 11.6 on ARM M1 macbook
```

### Terraform Version

```
Terraform v1.0.8
```

### Steps To Reproduce
Right after installing the extension the following error message is reported:

```
Unable to launch language server: ENOENT: no such file or directory, access '/Users/cezarsa/.vscode/extensions/hashicorp.terraform-2.15.0/lsp/terraform-ls'undefined
```
<img width="497" alt="Screen Shot 2021-10-05 at 14 44 27" src="https://user-images.githubusercontent.com/11041/136076110-de0878fc-e833-4a4c-93b2-ba79e7e47285.png">

The same error message shows up if I try to run `Terraform: Enable Language Server` command.

### Expected Behavior
The language server executable should be present and the language server should start.

### Actual Behavior
Only the error message above is reported.
 
### Additional context
There's no `lsp` directory in the extension installation path:
```
$ ls /Users/cezarsa/.vscode/extensions/hashicorp.terraform-2.15.0/
CHANGELOG.md                 README.md                    node_modules/                package.json                 syntaxes/                    terraform.png
LICENSE.txt                  language-configuration.json  out/                         snippets/                    terraform-banner.png         testFixture/
```

**Comment by jpogran** [comment]
I'm glad you go it working with a custom install. In https://github.com/hashicorp/vscode-terraform/issues/812 we implemented a different way of installing the terraform-ls binary to avoid file permission issues, which may work for you. We continue to work on bundling the terraform-ls binary inside the extension in https://github.com/hashicorp/vscode-terraform/issues/820. 

https://github.com/hashicorp/vscode-terraform/pull/868 might be of interest to you, as we configured the extension to silently install the terraform-ls binary without prompting the user. This is currently released in the [preview extension](https://marketplace.visualstudio.com/items?itemName=HashiCorp.terraform-preview) if you would like to try this out now.
